### PR TITLE
Add input

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

File: components/atoms/input/input.loading.tsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export function InputLoading() {
+  return <div />;
```
Comment (haydencleary) [inline components/atoms/input/input.loading.tsx:2]: No skeleton ?

File: components/atoms/input/input.core.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      }}
+      label={label}
+      variant="bordered"
+      labelPlacement="outside-left"
+      startContent={startContent}
+      endContent={endContent}
+      isDisabled={disabled}
+      disabled={disabled}
+      onValueChange={handleChange}
+      onChange={onChange}
```
Comment (haydencleary) [inline components/atoms/input/input.core.tsx:46]: I don't think we need both, just the `onChange` should do.

File: components/atoms/input/input.core.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        helperWrapper: cn(slots.helperWrapper(), classNames?.helperWrapper),
+        description: cn(slots.description(), classNames?.description),
+      }}
+      label={label}
+      variant="bordered"
+      labelPlacement="outside-left"
+      startContent={startContent}
+      endContent={endContent}
+      isDisabled={disabled}
+      disabled={disabled}
```
Comment (haydencleary) [inline components/atoms/input/input.core.tsx:44]: Both necessary ?

File: components/atoms/input/input.core.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+import { cn } from "src/utils/cn";
+
+import { TInputProps } from "./input.types";
+import { InputCoreVariants } from "./input.variants";
+
+export function InputCore({
+  classNames,
+  isError,
+  disabled,
```
Comment (haydencleary) [inline components/atoms/input/input.core.tsx:11]: I think it should be either `error` and `disabled` or `isError` and `isDisabled`. Slight preference for the prefixed version.

**Review by haydencleary — CHANGES_REQUESTED** [review]

File: components/atoms/input/input.core.tsx (hunk 3 above)
Comment (alexbeno) [reply components/atoms/input/input.core.tsx:44]: Next ui api.. :/

File: components/atoms/input/input.core.tsx (hunk 4 above)
Comment (alexbeno) [reply components/atoms/input/input.core.tsx:11]: So isError and isDisabled ?

**Review by haydencleary — APPROVED** [review]

**Pull request merged**